trino/sqlalchemy/compiler.py

```diff
@@ -102,6 +112,38 @@ def limit_clause(self, select, **kw):
             text += "\nLIMIT " + self.process(select._limit_clause, **kw)
         return text
 
+    def visit_column(self, column, add_to_result_map=None, include_table=True, **kwargs):
+        sql = super(TrinoSQLCompiler, self).visit_column(
+            column, add_to_result_map, include_table, **kwargs
+        )
+        table = column.table
+        return self.add_catalog(sql, table)
```

Comment:
I understood the benefit for table, but not sure about column. Could you add a test case to show the usecase?

Reply:
I think you're right that it's not needed. Removed the column visitor.